bin/alluxio

```diff
@@ -11,7 +11,11 @@
 #
 
 function printUsage {
-  echo "Usage: alluxio COMMAND [GENERIC_COMMAND_OPTIONS] [COMMAND_ARGS]"
+  echo "Usage: alluxio [-jvmdebug] COMMAND [GENERIC_COMMAND_OPTIONS] [COMMAND_ARGS]"
+  echo
+  echo "Flag description:"
+  echo -e "  -jvmdebug     \t If this flag specified, the env variable ALLUXIO_USER_DEBUG_JAVA_OPTS will be added to jvm flags."
```

Comment:
Because the option is not only work for one or two generic command, it can work for all generic command if it will start jvm process. So let this option be the generic command's option is not suitable. 

Forther, the  -debug option before this PR cannot work for getConf like the following command.

```bash
$ alluxio getConf -debug
```

But it can work well now.